diff --git a/Makefile.in b/Makefile.in
index 66e5d02..da507a8 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -46,7 +46,7 @@ includedir = $(prefix)/include
 # so compilers besides gcc can find it by default.
 # If it is empty or not defined, termcap.h will only be installed in
 # includedir. 
-oldincludedir = /usr/include
+oldincludedir =
 
 # Directory in which to install the documentation info files.
 infodir = $(prefix)/info
